doc/operation-and-maintenance/Logging-&-monitoring.md

```diff
@@ -59,45 +59,36 @@ Additionally, WombatOAM provides interfaces to feed the data to other OAM tools
 
 For more information see: [WombatOAM](https://www.erlang-solutions.com/products/wombat-oam.html).
 
-### graphite-collectd
+### OS level metrics: graphite-collectd
 
-To monitor MongooseIM during load testing, we recommend the following open source applications:
+To monitor MongooseIM node during load testing, we recommend the following open source applications:
 
 - [Grafana](https://grafana.com/) is used for data presentation.
-- [Graphite](http://graphiteapp.org/) is a server used for metrics storage.
+- [Prometheus](https://prometheus.io) is a server used for metrics storage. Alternatively, MongooseIM also works with [Graphite](http://graphiteapp.org/).
 - [collectd](http://collectd.org/) is a daemon running on the monitored nodes capturing data related to CPU and Memory usage, IO etc.
 
-### Plug-in Exometer reporters
+### Application metrics
+
+MongooseIM uses [Prometheus](https://prometheus.io/) as the default metrics solution, but also supports Exometer with Graphite exporter.
+Enable the chosen one in the [instrumentation section](../configuration/instrumentation.md).
+
+#### Prometheus
+
+MongooseIM exposes metrics in the Prometheus format by default.
+More information about configuration can be found in [the HTTP listeners section](../listeners/listen-http.md#handler-types-prometheus-mongoose_prometheus_handler).
+
+#### Exometer
 
 MongooseIM uses [a fork of Exometer library](https://github.com/esl/exometer_core) for collecting metrics.
 Exometer has many plug-in reporters that can send metrics to external services. We maintain [exometer_report_graphite](https://github.com/esl/exometer_report_graphite) and [exometer_report_statsd](https://github.com/esl/exometer_report_statsd) for Graphite and StatsD respectively.
```

Comment:
Ok, so the problem is that we don't support StatsD anymore in instrumentation. We would need to add support. Maybe we need to add this functionality? But we would need to add some tests, as there are none. I think I could add a story for this.

Reply:
Yes, I've noticed this too as well, and wasn't sure if I should remove this line. I will try to reword the next sentence maybe to be less misleading.
As a sidenote, I think we should in general revisit this doc page in general and make sure it is still correct. I'd say maybe even change it to a tutorial along the lines of "How to set up metrics for MongooseIM?", with differences between Exometer/Prometheus explained and up to a Grafana dashboard. We have a task to create a Grafana dashboard for MIM, which could be completed beforehand.

Reply:
I reworded this paragraph here: https://github.com/esl/MongooseIM/commit/4447d7ef46e06c5a8100543942521a57f69a34af